### PR TITLE
Add cache of 2 minutes to URLs

### LINKED ISSUE #21: Limit Bitbot's URL title responses to once every N minutes
Prevent loops and link spam

### PR DESCRIPTION
**Pull request by sylviamoss**
Fixes #21

Adds the URL to the bbotdb with a timestamp in order to check if the URL were visited in the last two minutes.

### CODE REVIEW

File: bitbot/links.go [hunk 1]
```diff
@@ -26,8 +32,56 @@ func isURL(message string) bool {
 	return xurls.Strict.MatchString(message)
 }
 
+func wasLookedUpInTheLastMintues(url string) bool {
+	db, err := newDB()
```
Comment (bbriggs) [inline bitbot/links.go]: We already have a DB exposed to us as `b.DB`, so there is no need to create a new one.

Comment (bbriggs) [reply bitbot/links.go]: This should be a method attached to `Bot` so that it gets access to the shared DB instance spawned and available as `b.DB`

**Comment by bbriggs** [comment]
I'm out of town this week and will try to get to all the PRs and issues as soon as I can. Sorry for any delay.

**Comment by m-242** [comment]
We cannot merge yet as the Title in cache isn't returned.
Oh and the test don't pass, but merging #52 and #53 should take care of that.

**Comment by bbriggs** [comment]
#52 and #53 have been merged

**Comment by bbriggs** [comment]
We could merge this into a temporary local branch, fix it up and then get it into master.

**Comment by bbriggs** [comment]
This needs to be rebased and cause up. You want to take over getting this merged, @m-242?

**Comment by m-242** [comment]
gladly

**Comment by m-242** [comment]
Replaced by PR #66

**Pull request closed without merging**